.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: CI/CD
+on:
+  pull_request:
+    branches: [ main ]
```

Comment:
Does it mean: "All pull requests to `main`"?

Reply:
Yes, it will be run for each pull requests to `main`